<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="map">
       <attribute name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/map</normaloff>:/icons/32x32/map</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="map_layout"/>
      </widget>
      <widget class="QWidget" name="kingdoms">
       <attribute name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/castle</normaloff>:/icons/32x32/castle</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="k_layout"/>
      </widget>
      <widget class="QWidget" name="warriors">
       <attribute name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/fiche_perso</normaloff>:/icons/32x32/fiche_perso</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="w_layout"/>
      </widget>
      <widget class="QWidget" name="book">
       <attribute name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/book</normaloff>:/icons/32x32/book</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QHBoxLayout" name="b_layout"/>
      </widget>
      <widget class="QWidget" name="">
       <attribute name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/32x32/info</normaloff>:/icons/32x32/info</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Database</string>
    </property>
    <addaction name="actionGenerate_Thumbnail"/>
    <addaction name="actionLaunch_SQlite"/>
    <addaction name="actionReset_attributes"/>
    <addaction name="actionAdd_Kingdom"/>
   </widget>
   <widget class="QMenu" name="menuDesign">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionStylesheet"/>
    <addaction name="actionApplyStyleSheet"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuTools"/>
   <addaction name="menuDesign"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="profilDockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
   </property>
   <property name="windowTitle">
    <string>Profil</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="profil_content"/>
  </widget>
  <action name="actionHome">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/24x24/home</normaloff>:/icons/24x24/home</iconset>
   </property>
   <property name="text">
    <string>home</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/24x24/quit</normaloff>:/icons/24x24/quit</iconset>
   </property>
   <property name="text">
    <string>quit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/24x24/save</normaloff>:/icons/24x24/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save current modifications</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionGenerate_Thumbnail">
   <property name="text">
    <string>Generate Thumbnail</string>
   </property>
  </action>
  <action name="actionLaunch_SQlite">
   <property name="text">
    <string>Launch SQlite</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/24x24/settings</normaloff>:/icons/24x24/settings</iconset>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </action>
  <action name="actionReset_attributes">
   <property name="text">
    <string>Reset attributes</string>
   </property>
  </action>
  <action name="actionAdd_Kingdom">
   <property name="text">
    <string>Add Kingdom</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionStylesheet">
   <property name="text">
    <string>EditStyleSheet</string>
   </property>
  </action>
  <action name="actionApplyStyleSheet">
   <property name="text">
    <string>ApplyStyleSheet</string>
   </property>
  </action>
  <action name="actionSelection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/32x32/selected</normaloff>:/icons/32x32/selected</iconset>
   </property>
   <property name="text">
    <string>selection</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/32x32/filtered</normaloff>:/icons/32x32/filtered</iconset>
   </property>
   <property name="text">
    <string>filter</string>
   </property>
  </action>
  <action name="actionAll">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resources.qrc">
     <normaloff>:/icons/32x32/all</normaloff>:/icons/32x32/all</iconset>
   </property>
   <property name="text">
    <string>all</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
